collectors/ebpf.plugin/ebpf.c

```diff
@@ -1263,45 +1265,41 @@ static void ebpf_update_table_size()
 /**
  * Set Load mode
  *
- * @param load  default load mode.
+ * @param origin specify the configuration file loaded
  */
-static inline void ebpf_set_load_mode(netdata_ebpf_load_mode_t load)
+static inline void ebpf_set_load_mode(netdata_ebpf_load_mode_t load, netdata_ebpf_load_mode_t origin)
 {
-#ifdef LIBBPF_MAJOR_VERSION
-    if (load == EBPF_LOAD_CORE || load == EBPF_LOAD_PLAY_DICE) {
-        load = (!default_btf) ? EBPF_LOAD_LEGACY : EBPF_LOAD_CORE;
-    }
-#else
-    load = EBPF_LOAD_LEGACY;
-#endif
+    UNUSED(load);
```

Comment:
Why do you keep the parameter if you are not going to use it anymore?

Reply:
Good point. I push another commit modifying the function algorithm. 
I remember I kept this to change the algorithm like I did, but I forgot. :facepalm: 
Thanks!